docs/howto/environment.md

```diff
@@ -15,35 +15,41 @@
   ```
   For comparison devs have used node at 10.13.x and 12.x.x.
 
-3. Install development copies of cylc and cylc-uiserver:
+3. Install
+   [cylc/cylc-flow](https://github.com/cylc/cylc-flow/) and
+   [cylc/cylc-uiserver](https://github.com/cylc/cylc-uiserver/):
   ```bash
-  pip install -e $DEVDIR/cylc
-  pip install -e $DEVDIR/cylc-uiserver
+  pip install -e "${DEVDIR}/cylc-flow" -e "${DEVDIR}/cylc-uiserver"
```

Comment:
That will change soon. I think `master` branch of `cylc-uiserver` does not rely on `cylc-flow` being installed.

The branch of David that uses GraphQL needs `cylc-flow`, but not the alpha versions. It is installing his fork of cylc-flow with the related GraphQL backend work.

But I think it might still be interesting to install cylc-flow editable for development I think - though the developer may need to edit `setup.py` for the version range

Reply:
Noted. We should update when that happens.